src/theme.scss

```diff
@@ -41,6 +42,14 @@ mat-icon[mat-card-avatar] {
   line-height: 40px;
 }
 
+// Fix Icon sizes used in table avatars
+mat-icon[mat-table-avatar] {
```

Comment:
Maybe the table avatar is not the right thing to use? I haven't done the visual check yet